docs/root/version_history/current.rst

```diff
@@ -13,7 +13,7 @@ Bug Fixes
 ---------
 *Changes expected to improve the state of the world and are unlikely to have negative effects*
 
-* docker: update Docker images (``distroless`` -> ``49d2923f35d6``) to resolve CVE issues in container packages.
+* repo: fix version to resolve release issue.
```

Comment:
Clarify fix "version of" what? Seems ambiguous

Reply:
point taken - but i think we just want to get this out the door at this point